logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/AvsClient.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.call
+
+data class AvsClient(
+    val userId: String,
+    val clientId: String
+)
+
+data class AvsClientList(
+    val clients: List<AvsClient>
+) {
+    fun toJsonString(): String {
+        val start = "{"
+        var startClients = "clients:["
```

Comment:
Missing quotes around `clients`?

Should be something like `"\"clients\"":[` I think.

Reply:
Yup yup, changed it on your function suggestion!